pkg/storage/db.go

```diff
@@ -473,8 +475,125 @@ func (db *DB) Store(ctx context.Context, cloudAssetChanges domain.CloudAssetChan
 		}
 		return err
 	}
-	return tx.Commit()
+	if err = tx.Commit(); err != nil {
+		return err
+	}
+	ver, err := db.GetSchemaVersion(ctx)
+	if err != nil || ver < DualWriteSchemaVersion {
+		return nil
```

Comment:
question: is it desirable to not return the error here if `err` isn't null?

Reply:
although I get in the case of `ver < DualWriteSchemaVersion` being true err would be nil so we couldn't return anything. I just want to clarify that it's desirable to not return the error here.?

Reply:
the err is for getting the schema version, which is unrelated to the write to the old schema (which was successful at this point). 
The reason for error when getting schema version is - the database was rolled back to the state before we introduced versioning/migration ('ver 0'), which is a totally valid condition from "old" schema perspective (as we already wrote data to old schema) but does not allow us to do dual-write.